examples/aggregator/proto/aggregator_init.proto

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.examples.aggregator_init;
+
+import "aggregator.proto";
+import "oak_services/proto/grpc_invocation.proto";
+
+// Initialization message that should be sent to Aggregator Oak Node.
+message AggregatorInit {
+  // Channel for receiving invocations from gRPC server.
+  oak.invocation.GrpcInvocationReceiver grpc_server_invocation_receiver = 1;
+  // Channel for sending invocations to gRPC client that is connected to an external backend.
```

Comment:
The backend here is not a database, it could be some service that uses the aggregated data, but doesn't necessarily store it.
For now changing the comment to:
```
  // Channel for sending invocations to gRPC client that is connected to an external service that
  // expects aggregated data from Oak.
```